README.md

```diff
@@ -388,71 +647,427 @@ Open the `Student` class and add the `(fetch = FetchType.LAZY)` property to the
 `@OneToOne` annotation:
 
 ```java
+package org.example.model;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
 @Entity
 @Table(name = "STUDENT_DATA")
 public class Student {
-    @Id
-    @GeneratedValue
-    private int id;
+   @Id
+   @GeneratedValue
+   private int id;
 
-    private String name;
+   private String name;
 
-    @Temporal(TemporalType.DATE)
-    private Date dob;
+   private LocalDate dob;
 
-    @Enumerated(EnumType.STRING)
-    private StudentGroup studentGroup;
+   @Enumerated(EnumType.STRING)
+   private StudentGroup studentGroup;
 
-    @OneToOne(fetch = FetchType.LAZY)
-    private IdCard card;
-}
+   @OneToOne(fetch = FetchType.LAZY)
+   private IdCard card;
+
+   //getters/setters/toString
 ```
 
-A one-to-one association uses the `FetchType.EAGER` value if no value is
-specified. By setting it to `FetchType.LAZY`, we can defer the join query. Run
-the `main` method of the `JpaRead` class again to see how the Hibernate query
+A one-to-one association uses the `FetchType.EAGER` value as a default if no value is
+specified. By setting it to `FetchType.LAZY`, we can defer the join query
+until the `getCard()` method is called. 
```

Comment:
I think I might be reviewing these PRs out of order; if this lesson comes before the lesson about removing rows in a cascade fashion, ignore that comment on that lesson as I see the fetch is covered here.

Reply:
ok thanks